### PR TITLE
2017march newsletter: work in progress

### PR DESCRIPTION
**Pull request by imrehg**
* currently placeholder for the headliner 2.0 release, will be filled
* currently links to draft Soracom blogpost, that will be updated

### CODE REVIEW

File: src/data/ad.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "title": "OpenWhisk and resin.io for edge computing",
-  "text": "A pair of writeups for using resin.io with OpenWhisk as a distributed compute service",
-  "link": "https://forums.resin.io/t/openwhisk-writeup/593",
-  "image": "https://cdn-images-1.medium.com/max/1200/1*EEJjDyoHUnuY90zm2d3Uqw.png"
+  "title": "Changes in resin.io support",
```
Comment (alisondavis17) [inline src/data/ad.json]: edit: changes _to_ resin.io support

File: src/data/ad.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "text": "Our goal is for the entire resin.io community to become more knowledgeable with each issue solved and question answered.",
```
Comment (alisondavis17) [inline src/data/ad.json]: let's preview all of the first paragraph from the forum post, so it's clear what the change actually is: _"We’re making a few changes to our support process. Our goal is for the entire resin.io community to become more knowledgeable with each issue solved and question answered. We’re therefore shifting our support focus to the place with the most impact and visibility: our forums."_

Also, can/should we also add a second link to resin.io/support?

Comment (imrehg) [inline src/data/ad.json]: @alisondavis17 I've added all of that in, and it looks pretty big for a preview, but not crazy. In the current formatting all the text is a link to the post, so can't really add extra links.

I'm guessing if we wanna include that link, it should also rather be in that post itself. It was, but not explicitly. I've added an expanded note there, let me know if it's okay:

>  (If you’re not a paid user but would like to become one, check our [support plans](https://resin.io/support) or contact us at sales@resin.io). 

What do you think?

Comment (alisondavis17) [inline src/data/ad.json]: Sounds good on all fronts!

**Pull request merged**